User-Help/NewUserBlurb.md

```diff
@@ -0,0 +1,11 @@
+<!--- Local Intro Blurb -->
```

Comment:
Suggestion: <!--- Local Intro Blurb: Example (adjust for your community) -->

Reply:
Sorry, your suggestion isn't coming through? not sure if I changed something it was referring to in the latest commit.

Reply:
Argh -- I didn't notice that my suggested HTML comment got swallowed.  I suggested expanding the comment text to "Local Intro Blurb: Example (adjust for your community)" to make it overly clear that this is an example.

Reply:
Oooh, I like that. Let me update.